modelCard.yml

```diff
@@ -0,0 +1,20 @@
+- name: laituan245/molt5-large-smiles2caption
```

Comment:
please provide this information in the body of your PR, not as a file next time.

Reply:
Reference is supposed to be a single string, so please provide link to the paper or a DOI directly, if there isn't one, then the github repo or any other link would be fine.

Reply:
> please provide this information in the body of your PR, not as a file next time.

I can delete this file and then edit the PR. Would this be better?

Reply:
Yes, thanks @dessygil  !